site/nav.yml

```diff
@@ -49,6 +49,7 @@ nav:
     - Multi-engine support: multi-engine-support.md
     - How to release: how-to-release.md
     - Terms: terms.md
+    - Status: status.md
```

Comment:
"Status" is too general and unclear on what this page is about.

Reply:
Looks good. I'd also suggest putting it next to `AES GCM Stream spec ` since implementations are closely related to specs.